components/clientComponents/forms/Review/reviewUtils.ts

```diff
@@ -0,0 +1,208 @@
+import { AddressComponents, FileInputResponse, FormElement, FormElementTypes } from "@lib/types";
```

Comment:
Note to self: the goal of this utils file is to use as a placeholder and in a followup PR move much/most of the below to general utils. Ideally this file will be deleted in the future.
The below code was used in ways it was never mean to be and the original should have been refactored at that time IMO.